examples/basic/README.md

```diff
@@ -1,5 +1,30 @@
 # Basic example
 
-A simple example with 3 servers and a load balancer that distributes the traffic between them. The example assumes that each of the servers has some application listening on port 3000.
+A simple example with 3 servers and a load balancer that distributes the traffic between them. The example uses an user-data script to launch a HTTP server listening to port 80 on each server. The example does not enable TLS on frontend by default.
 
-Once you deploy it, go to your domain settings and add a CNAME record that points to the `lb_url` output variable. After the change gets propagated, all the traffic to your domain (`my.domain.com` in this example) will be distributed among the 3 servers.
+## Getting started
+
+To deploy private network, three servers, and load balancer on your UpCloud account, run:
+
+```bash
+terraform init
+terraform apply
+```
+
+Once the deployment is complete and load balancer has reached `running` state you can use `curl` or your browser to send request to the load balancer available in the URL defined by `lb_url` output variable:
```

Comment:
We could mention here that it might take some time to DNS name to propagate which may result in error:
`curl: (6) Could not resolve host: ....`

Reply:
Good idea! I added note about this